client/my-sites/upgrades/domain-management/components/icann-verification-card.jsx

```diff
@@ -15,15 +15,25 @@ let IcannVerificationCard = React.createClass( {
 		selectedDomainName: React.PropTypes.string.isRequired
 	},
 
+	getExplanation() {
+		switch ( this.props.explanationContext ) {
+			case 'name-servers':
+				return this.translate(
+						'You must verify your email address through the ICANN ' +
+						'verification email before you are able to update the name ' +
+						'servers for your domain.' );
+		}
+
+		return this.translate(
```

Comment:
What is the reason for not having this `return` statement in the `switch` under `default:`?